eng/common/pipelines/templates/steps/check-spelling.yml

```diff
@@ -14,13 +16,25 @@ parameters:
 
 steps:
   - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: 16.x
+      displayName: Use Node.js 16.x
+
+    # NPM version installed on some machines is very outdated and does not
+    # support `npm exec`. In some cases it may be necessary to revert the NPM
+    # upgrade after the spell check has finished by running something like:
+    # `npm install -g npm@6.14.15`
+    - pwsh: npm install -g npm
```

Comment:
I think you should pin npm too like you're doing with node.

Reply:
We're using `npx` instead of messing with the npm installed on the machine.